members/README.md

```diff
@@ -2,25 +2,7 @@
 
 ## Member object
 
-```
-{
-  'id': string,
-  'first_name': string,
-  'last_name': string,
-  'yoe': number,
-  'company': string,,
-  'designation': string,
-  'img': string,
-  'github_id': string,
-  'linkedin_id': string,
-  'twitter_id': string,
-  'instagram_id': string,
-  'site': string,
-  'github_display_name': string,
-  'isMember': boolean,
-  'tokens': {}
-}
-```
+same as the [user's](../users) object
```

Comment:
Can we link the user object instead? `https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#user-object` 

Can we capitalize `same`? Also, better to call it `user object` than `user's object`?

Reply:
https://github.com/Real-Dev-Squad/website-api-contracts/pull/32/commits/addc5eca0c0937713c79bfc5751950d3e66f6c17